cni/azure-linux.conflist

```diff
@@ -6,7 +6,6 @@
          "type":"azure-vnet",
          "mode":"bridge",
          "bridge":"azure0",
-         "disableHairpinOnHostInterface":true,
```

Comment:
Will we get duplicate UDP/Tcp packets from the load balancers like we did before? https://github.com/Azure/azure-container-networking/pull/530

Reply:
yes we cannot support UDP.